common/djangoapps/entitlements/models.py

```diff
@@ -452,3 +452,16 @@ def __unicode__(self):
             self.support_user,
             self.reason,
         )
+
+    @classmethod
+    def get_support_reasons_list(cls):
+        """
+        Method for retrieving a serializable version of the entitlement support reasons
+
+        Returns Array of reasons
```

Comment:
In python we use the form 

```
Returns
    list: Containing the reasons
```

Reference with more detail

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings